test/jdk/java/net/spi/URLStreamHandlerProvider/Basic.java

```diff
@@ -125,6 +132,16 @@ static void viaBadProvider(String protocol, Consumer<Result> resultChecker,
                                 sysProps);
     }
 
+    static void viaCircularProvider(
+            String protocol, Consumer<Result> resultChecker, String... sysProps)
+            throws Exception {
+        viaProviderWithTemplate(
+                protocol,
+                resultChecker,
+                TEST_SRC.resolve("circular.provider.template"),
+                sysProps);
+    }
```

Comment:
I assume you've re-format the changes to Basic.java to be consistent with this existing code in the test (looks a bit strange on first sighting to see a completely different style being introduced).